subliminal/refiners/tmdb.py

```diff
@@ -0,0 +1,335 @@
+"""Refine the :class:`~subliminal.video.Video` object by searching on TMDB."""
+
+from __future__ import annotations
+
+import logging
+import re
+from typing import Any, ClassVar, cast
+
+import requests
+
+from subliminal import __short_version__
+from subliminal.cache import REFINER_EXPIRATION_TIME, region
+from subliminal.utils import sanitize
+from subliminal.video import Episode, Movie, Video
+
+logger = logging.getLogger(__name__)
+
+series_re = re.compile(r'^(?P<series>.*?)(?: \((?:(?P<year>\d{4})|(?P<country>[A-Z]{2}))\))?$')
+
+#: TMDB subliminal API key
+TMDB_API_KEY = '3dac925d5d494853ea6ef9161011fbb3'
```

Comment:
what's the quota on this API key?

Reply:
I understood it doesn't have quota: https://developer.themoviedb.org/docs/rate-limiting

But it's a good point in general, should we provide a default API key. It also applies to the OMDB and TVDB refiners.